release-team/role-handbooks/branch-manager/README.md

```diff
@@ -177,6 +196,9 @@ Releasing has multiple phases.  In the first phase the non-public build artifact
 * `./release-notify v1.12.0-alpha.1`
 * `./release-notify v1.12.0-alpha.1 --nomock --mailto=${{YOUREMAILADDRESS}}[a][b]`
 
+**Note**
```

Comment:
nit: isn't this the same as already in https://github.com/kubernetes/sig-release/pull/619/files#diff-52866a7ca0c8666623044eeb11e59015R185 ?

Reply:
I feel this too, let's drop it for now.